lang/cs/Org.Apache.REEF.Tests/Functional/ReefFunctionalTest.cs

```diff
@@ -155,7 +155,7 @@ public void Dispose()
             CleanUp();
         }
 
-        protected void ValidateSuccessForLocalRuntime(int numberOfContextsToClose, int numberOfTasksToFail = 0, int numberOfEvaluatorsToFail = 0, string testFolder = DefaultRuntimeFolder, int retryCount = 60)
+        protected void ValidateSuccessForLocalRuntime(int? numberOfContextsToClose, int numberOfTasksToFail = 0, int numberOfEvaluatorsToFail = 0, string testFolder = DefaultRuntimeFolder, int retryCount = 60, string expectedCancellationMessage = null)
```

Comment:
This method is mainly for successful cases. If you only want to validate a message, you can just read from log file directly and validate in your test, keeping this method unchanged.